### PR TITLE
Add pipeline

### PR DESCRIPTION
**Pull request by yusif-ifraimov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,11 +213,24 @@ Also, there is a shortened version of the raw data - the "SP500" csv file. Since
...
+
+Speaking of analyzing the raw visual data,  we had to create a certain pipeline to put it into the proper format and then make forecasts based on that. First of all, simply inputting the whole sequence of candle plots and asking the models to make a forecast -(continue the picture) is not the best approach for our specific task. We need to detach the individual candles in the first place, remove the background noise (for instance the grid) and so on. The illustration below demonstrates how we do it. 
+
+![Pipeline](images/pipeline_ps.png)
 
 
+
+1. We take the  images with candle plots and then pass them trough our **selector**. The purpose of the **selector** is to find and detach each individual candle into separate images. 
+2. As an output we get separated candles. 
+3. Afterwards, we pass each individual candle trough **interpreter**. The purpose of interpreter is to convert  individual candles into more convenient numeric representation which we have already mentioned.
```
Comment (IliaOzhmegov) [inline README.md:227]: interpretatOr
that should be with a typo

File: README.md [hunk 2]
```diff
@@ -212,11 +213,24 @@ Also, there is a shortened version of the raw data - the "SP500" csv file. Since
...
+
+![Pipeline](images/pipeline_ps.png)
 
 
+
+1. We take the  images with candle plots and then pass them trough our **selector**. The purpose of the **selector** is to find and detach each individual candle into separate images. 
+2. As an output we get separated candles. 
+3. Afterwards, we pass each individual candle trough **interpreter**. The purpose of interpreter is to convert  individual candles into more convenient numeric representation which we have already mentioned. 
+   * **HLOC** - refers to High, Low, Open, and Close parameters of the candle. 
+4.  Afterwards we get the **HLOC** (High, Low, Open, and Close) values of our candles. The variable **__t__** represents the sequence of the candle. In other words the order. The index **__k__** - is the total number of candles at the input. It is very important to keep the sequence of the candles although we separate them as each next candle is dependent on previous one.
```
Comment (IliaOzhmegov) [inline README.md:228]: I don't think it look good to explain HLOC twice or more

File: README.md [hunk 3]
```diff
@@ -157,19 +158,19 @@ GOAL: It is a well spread to suffer. Get used to it!
...
+   2. The moving average MA(q) Model. | DONE
+   3. The ARMA(p,q) Model. | DONE
+   4. Maybe show partial autocorrelation function. | DONE
 5. Apply a NN:
-    1. Apply a time delay neural networks TDNN.
-    2. Apply a simple recurrent neural network RNN.
-    3. Apply a LSTM.
+   1. Apply a time delay neural networks TDNN.
+   2. Apply a simple recurrent neural network RNN.
+   3. Apply a LSTM.
```
Comment (IliaOzhmegov) [inline README.md]: That's all done, also add CNN

File: README.md (hunk 1 above)
Comment (yusif-ifraimov) [reply README.md:227]: Typora shows a mistake when I type intepretator. maybe I should update the pic then?

**Review by IliaOzhmegov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (IliaOzhmegov) [reply README.md:227]: that would be gret!

**Review by IliaOzhmegov — APPROVED** [review]

**Pull request merged**